mssql/deploy.sh

```diff
@@ -0,0 +1,12 @@
+echo "load env.."
+. env.sh
+bash ../remove.sh "$CONTAINER_NAME"
+sudo docker run --detach \
+    --name "$CONTAINER_NAME" \
+    --env=MSSQL_SA_PASSWORD="$MSSQL_SA_PASSWORD" \
+    --env=MSSQL_PID="$MSSQL_PID" \
```

Comment:
why we set sql server pid(process id)  manual ? it must be provided by OS. Or what `MSSQL_PID` means ?

Reply:
you can reach what it is for from this link https://learn.microsoft.com/en-us/sql/linux/sql-server-linux-configure-environment-variables?view=sql-server-ver16

Reply:
The pid you see here shows the edition or product key, not the process id.

Reply:
resolved